test/onnx/test_fx_op_consistency.py

```diff
@@ -137,7 +138,9 @@
         "scatter_add",
         "scatter_reduce",
         "square",
+        "sum",
         "unflatten",
+        # "var_mean",  # Segfault during onnx shape inference. Need to bump onnx version.
```

Comment:
Is there a issue to track it? or at least add a `TODO:`

Reply:
Sure #105073